WordPress/gutenberg
# Font Library: singularize install font families endpoint

## What?
- Singularize install font families endpoint. 
- Update frontend code to send only one per font family per request.
- Update JS and PHP unit tests accordingly.

## Why?
To make the `POST /font-families/` endpoint accept only one font family per request instead of many.

This is part of the effort around [refactoring the Font Library API.](https://github.com/WordPress/gutenberg/issues/55278)

## How?
Removing the capability to install several font families using only one request.

## Testing Instructions
- Run JS unit tests
- Run PHP unit tests
- Use the feature to install Google and Local fonts and check that is working as expected.
